docs/datatypes/io.md

```diff
@@ -161,3 +161,27 @@ val action: Task[String] = Task.effectTotal(i += 1) *> Task.fail(new Throwable("
 val cleanupAction: UIO[Unit] = UIO.effectTotal(i -= 1)
 val composite = action.ensuring(cleanupAction)
 ```
+### A full working example on using brackets
+```scala
```

Comment:
When using the `scala` lang directive, the snippet does not get typechecked, so we can't maintain its correctness over time.

If you switch this to `scala mdoc:silent` as the other snippets above, it'll be typechecked.